src/overlay/Peer.h

```diff
@@ -53,8 +53,32 @@ class Peer : public std::enable_shared_from_this<Peer>,
 {
 
   public:
+    static constexpr uint32_t FIRST_VERSION_SUPPORTING_FLOW_CONTROL = 20;
+
     typedef std::shared_ptr<Peer> pointer;
 
+    // Data structure that records the time it takes to process a message and
+    // sends the signal back to the peer. Callers must manually call
+    // `finishProcessing`.
```

Comment:
I'm so sorry to have noticed this, but: this comment and the two below still mention `finishProcessing` which is no longer the name of the method.